### PR TITLE
Add pr exists label

### PR DESCRIPTION
**Pull request by neonichu**
Mostly adds a "pr exists" label. This is supposed to mark issues for which an unmerged PR already exists. This both helps us to determine which open issues to look at, as well as users to look for that PR link / comment somewhere in the issue.

Also now optionally reads the GH API token from an environment variable for those of us who don't want extra files :)

Finally, adds two labels which were apparently added manually at some point.

### CODE REVIEW

**Comment by segiddins** [comment]
I think S4 already describes this?

**Comment by neonichu** [comment]
Does it? I'm actually not sure what many of the status labels mean — do we have a definition somewhere? If not, maybe we should create one.

**Comment by kylef** [comment]
@neonichu see https://guides.cocoapods.org/contributing/tickets-handling.html

**Comment by kylef** [comment]
I think the "waiting validation" tag is exactly what you're looking for, and this PR should be closed.

screen shot 2015-08-10 at 16 40 42

File: scripts/setup-github-labels.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 require 'octokit'
 
 token_file = File.expand_path("../../.github_access_token", File.dirname(__FILE__))
-token = File.file?(token_file) ? File.read(token_file).strip : ''
+token = File.file?(token_file) ? File.read(token_file).strip : ENV['GITHUB_API_TOKEN']
```
Comment (kylef) [inline scripts/setup-github-labels.rb]: This should be a separate PR... (especially since this PR may get closed)

**Comment by neonichu** [comment]
Ah, great, thanks — I somehow missed this. Will do a guides PR to bring that page in sync with reality (some label names are actually different now).

What's everyone's opinion on the "frameworks" and "plugin idea" labels? They're undocumented and seem to fall out of the structure described by the guide, so let's delete them?

**Comment by segiddins** [comment]
I'd be in favor of getting rid of them -- I don't like topic labels, but prefer the issue title to have them
